Tactility/Include/Tactility/settings/SystemSettings.h

```diff
@@ -1,12 +1,14 @@
 #pragma once
 
 #include "Language.h"
+#include <string>
 
 namespace tt::settings {
 
 struct SystemSettings {
     Language language;
     bool timeFormat24h;
+    std::string timeZoneCode;
```

Comment:
The changes in this file can be removed for now, as it's redundant with the current timezone saving.